Makefile

```diff
@@ -19,7 +19,7 @@ require-version:
 	if [ -z ${VERSION} ]; then echo "Need to set VERSION" && exit 1; fi;
 
 test-coverage:
-	go test --race -coverprofile=coverage.txt -covermode=atomic ./...
+	go test --race -coverprofile=coverage.txt -covermode=atomic -coverpkg=./... ./...
```

Comment:
Just to make sure I didn't misunderstand this, if test in one package covered code in another package, the indirect coverage should still be reported as covered overall right?

It seems to be changed in https://github.com/golang/go/commit/50455385b0e668656cac03d3012e48e071df6aa4 for the reason of consistency when testing individual packages but I don't believe that is the behavior we want for an overall coverage report.

Reply:
:+1: